op-node/rollup/finality/plasma_test.go

```diff
@@ -23,165 +38,196 @@ func (b *fakePlasmaBackend) OnFinalizedHeadSignal(f plasma.HeadSignalFn) {
 
 var _ PlasmaBackend = (*fakePlasmaBackend)(nil)
 
-// TODO: Fix this test
-// func TestPlasmaFinalityData(t *testing.T) {
-// 	logger := testlog.Logger(t, log.LevelInfo)
-// 	l1F := &testutils.MockL1Source{}
-// 	l2F := &testutils.MockL2Client{}
```

Comment:
seems commenting out large chunk of code is not a good idea. make PR review much harder when we need to uncomment it back lol